csharp/src/Drivers/Apache/Spark/SparkParameters.cs

```diff
@@ -32,6 +32,7 @@ public static class SparkParameters
         public const string Type = "adbc.spark.type";
         public const string DataTypeConv = "adbc.spark.data_type_conv";
         public const string TLSOptions = "adbc.spark.tls_options";
+        public const string RequestTimeoutMilliseconds = "adbc.spark.request_timeout_ms";
```

Comment:
Inline with the property name, I wonder if the setting name should also specify this is for HTTP?

Reply:
Updated properties and options to indicate its use with the HTTP type connection.